src/kirby/components/list/list.component.scss

```diff
@@ -36,8 +36,20 @@ $shadow: get-elevation(2);
       margin-bottom: $outline-width;
       background-color: $background-color-hover;
     }
-  }
 
+    $list-colors: ('white', 'light');
+    @each $color-name, $color-value in $list-colors {
+       &.#{$color-name} {
+               background-color: #{get-color($color-name)};
```

Comment:
Don't we also need to add the text color?
`color:  #{get-color($color-name+'-contrast')};`

Reply:
Yes (y)